tests/e2e/e2e_certmanager_test.go

```diff
@@ -17,9 +18,12 @@ var _ = OSMDescribe("1 Client pod -> 1 Server pod test using cert-manager",
 	},
 	func() {
 		Context("CertManagerSimpleClientServer", func() {
-			const sourceNs = "client"
-			const destNs = "server"
-			var ns []string = []string{sourceNs, destNs}
+			var (
+				clientNamespace     = framework.RandomName()
+				serverNamespace     = framework.RandomName()
```

Comment:
This could make troubleshooting and investigating a failing test more difficult. Instead of knowing which namespace has which component, the namespaces now will be random strings. Should we perhaps prefix this so that we should be able to distinguish between source/client and destination/server namespaces?